cli/tests/deno_test_only.ts

```diff
@@ -0,0 +1,15 @@
+Deno.test({
+  name: "abc",
+  fn() {},
+});
+
+Deno.test({
+  only: true,
+  name: "def",
+  fn() {},
+});
```

Comment:
Out of curiosity what happens when you do:
```
Deno.test({
  only: true,
  ignore: true,
  name: "def",
  fn() {},
});
```

Reply:
```
running 1 tests
test def ... ignored (0ms)

test result: ok. 0 passed; 0 failed; 1 ignored; 0 measured; 0 filtered out (1ms)

FAILED because the "only" option was used
```
First the `only` filter is applied (without affecting a metric), then the `--filter` filter is applied (increases the `filtered out` metric), then the `ignore` filter is applied (increases the `ignored` metric and is listed as `ignored`).